jormungandr/src/rest/v1/mod.rs

```diff
@@ -12,10 +12,9 @@ pub fn filter(
     let root = warp::path!("v1" / ..);
 
     let fragments = {
-        let root = warp::path!("fragments");
+        let root = warp::path!("fragments" / ..);
 
         let post = warp::path::end()
-            .and(warp::post())
```

Comment:
Why remove the post filter? It shouldn't work at all.

Reply:
you're right ! sorry